cmd/gateway/azure/gateway-azure.go

```diff
@@ -342,17 +342,11 @@ func azureToObjectError(err error, params ...string) error {
 
 // mustGetAzureUploadID - returns new upload ID which is hex encoded 8 bytes random value.
 func mustGetAzureUploadID() string {
-	var id [8]byte
-
-	n, err := io.ReadFull(rand.Reader, id[:])
+	uuid, err := uuid.New()
 	if err != nil {
-		panic(fmt.Errorf("unable to generate upload ID for azure. %s", err))
-	}
-	if n != len(id) {
-		panic(fmt.Errorf("insufficient random data (expected: %d, read: %d)", len(id), n))
+		panic(err)
```

Comment:
We have `logger.FatalIf` with the new logger package.
I think  we need to use `logger.FatalIf` going forward.